r2/r2/controllers/front.py

```diff
@@ -879,7 +879,8 @@ def GET_about(self):
         Data includes the subscriber count, description, and header image."""
         if not is_api() or isinstance(c.site, FakeSubreddit):
             return self.abort404()
-        item = Wrapped(c.site, accounts_active_count=c.site.accounts_active)
+        item = Wrapped(c.site, author=c.site.author_slow,
+                       accounts_active_count=c.site.accounts_active)
```

Comment:
This is the only place that the attribute is missing and needs to be added. In all other places (/subreddits listings), the builder takes care of it.